assets/js/membraneWebRTC.ts

```diff
@@ -384,7 +375,7 @@ export class MembraneWebRTC {
   /**
    * Replaces a track that is being sent to the SFU server.
    * @param track - Audio or video track.
-   * @param {MediaStreamTrack} oldTrack - Audio or video track to replace.
+   * @param {string} trackId - Audio or video track to replace.
```

Comment:
```suggestion
   * @param {string} trackId - Id of audio or video track to replace.
 ```  
 
 Also we need to fix example code for this function